src/appleseed/foundation/image/color.h

```diff
@@ -1183,4 +1186,13 @@ Color<T, 3> integer_to_color3(const Int i)
         static_cast<T>(z) * (1.0f / 4294967295.0f));
 }
 
+template <typename T>
+T compute_error_squared(const Color<T, 3>& c1, const Color<T, 3>& c2)
```

Comment:
Mark inline.

Reply:
I'm admittedly not clear on why we want this `inline`d, though this is what I worked through
- This function is used in multiple cpp files and it compiles okay (not necessarily a good reason not to `inline` though)
- Perhaps we really want to hint to the compiler that code should be generated inline for this to reduce function call overhead since it is called within expensive loops

Reply:
> perhaps we want to hint to the compiler that code should be generated inline for this to reduce function call overhead since it is called within expensive loops.

That's exactly the reason, although admittedly I'm not sure that any compiler actually takes the hint seriously anymore.